e2e/Dockerfile.tfserver

```diff
@@ -0,0 +1,8 @@
+#FROM tensorflow/tf_grpc_test_server:ccbc039fbe5a
+FROM tensorflow/tf_grpc_test_server:latest
+
```

Comment:
Do we need our own custom gRPC server for parameter servers? If the code is using the tf.Estimator API and you call  train_and_evaluate
https://www.tensorflow.org/api_docs/python/tf/estimator/train_and_evaluate

I think that will automatically start PS as needed based on TF_CONFIG
This [line](https://github.com/tensorflow/tensorflow/blob/7cabd979a46febafdb90a83865cd743233d4449f/tensorflow/python/estimator/training.py#L541)

which calls [run_ps](https://github.com/tensorflow/tensorflow/blob/7cabd979a46febafdb90a83865cd743233d4449f/tensorflow/python/estimator/training.py#L614)

Reply:
Yeah this is confusing, and I ended up not using the standard grpc server. I think I'm going to strip out references to it other than as an initial base image for the model.